help_to_heat/frontdoor/routing/forwards_routing.py

```diff
@@ -405,13 +410,24 @@ def _council_tax_band_next_page(answers):
     return _post_council_tax_band_next_page(answers)
 
 
-# confirms epc if it was found
+# confirms epc if it was found, or send to no epc
 def _post_council_tax_band_next_page(answers):
     epc_found = answers.get(epc_found_field)
+    address_choice = answers.get(address_choice_field)
+    epc_select_choice = answers.get(epc_select_choice_field)
+    address_select_choice = answers.get(address_select_choice_field)
     if epc_found == field_yes:
         return epc_page
     if epc_found == field_no:
-        return _post_epc_next_page(answers)
+        # if they entered an address manually, don't show the no epc page as an epc wasn't searched for
+        if address_choice == address_choice_field_enter_manually:
+            return _post_epc_next_page(answers)
+        if epc_select_choice == epc_select_choice_field_enter_manually:
+            return _post_epc_next_page(answers)
+        if address_select_choice == address_choice_field_enter_manually:
+            return _post_epc_next_page(answers)
```

Comment:
```suggestion
        if address_choice == address_choice_field_enter_manually
          or epc_select_choice == epc_select_choice_field_enter_manually
          or address_select_choice == address_choice_field_enter_manually:
            return _post_epc_next_page(answers)
```
or however black wants it formatted

Maybe it could be made clearer here why there's three different routes into this?

Reply:
have reworked this a bit. can touch on the general question in our next catchup perhaps?

Reply:
Sure